### ISSUE TITLE
Examples in main do not produce graphical output anymore

### ISSUE THREAD

**Issue by ggrammel** [Telecominfraproject/gnpy#67]
trying out the Examples (after having applied workaround reported before on the README file) it turns out that the graphical output is not available anymore. The previous master did produce a window with a graphical representation of the network. The current master only produces text output.

**Comment by jeanluc-auge** [comment]
You are right, it's still there and it's just been put in comment [here](https://github.com/Telecominfraproject/gnpy/blob/master/examples/transmission_main_example.py#L112) and in a [def](https://github.com/Telecominfraproject/gnpy/blob/master/examples/transmission_main_example.py#L42)  because:
* the graphical network representation requires the location coordinates in the metadata
* if this information is not present the code crashes
* with all the new network topology examples, there is no coordinates and I didn't want that to stop us
=> so we just need #TODO modify the [def](https://github.com/Telecominfraproject/gnpy/blob/master/examples/transmission_main_example.py#L42) so that it accepts metadata with 0.0 coordinates

**Comment by ggrammel** [comment]
the graphical representation is certainly not something that should block the development. I would see it as a useful extension that helps to understand the examples we've put out. Or in other words: examples with coordinates should (optionally) produce graphical output. We may probably want to use a switch to indicate if graphical output is desired and describe our examples accordingly.
